charts/hlf-k8s/templates/deployment-enrollement-operator.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       serviceAccountName: {{ template "substra.fullname" . }}
       containers:
         - name: fabric-tools
-          image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+          image: {{ .Values.images.hlfk8sca.repository }}:{{ .Values.images.hlfk8sca.tag }}
```

Comment:
I find this line confusing, reading the code quickly we could think that we are using a CA image for the enrollment.
We could use something like `hlfk8sCaClient`.

Reply:
I agree for `hlfk8sCaEnrollment` as it is only used for CA enrollement. Nevertheless `hlfk8s` image is used for multiple other deployment (for instance toolbox)

Reply:
Yes, there is no issue with `hlfk8s`, reading the code we can clearly see it is a multi-purpose image with tooling.

Just for information, why did you choose to use a different image here and not add more tools to the hlfk8s image ?

Reply:
Because, the two images are not based on the same version (fabric 2.2 and 1.4.9)
CA does not have a version increase with HLF 2.x